src/Main.java

```diff
@@ -15,21 +17,43 @@ public static void main(String[] args) {
         Wheel chevroletWheels = new Wheel(13, true);
         Wheel fordWheels = new Wheel(14, false);
         Wheel audiWheels = new Wheel(16, true);
+        Tank mazdaTank = new Tank(100);
+        Tank chevroletTank = new Tank(259);
+        Tank fordTank = new Tank(150);
+        Tank audiTank = new Tank(120);
         CarManager carManager = new CarManager();
-        carManager.carList.add(new Car("Mazda", mazdaEngine, blackBody, multifunctionedSteeringWheel, mazdaWheels));
-        carManager.carList.add(new Car("Chevrolet", chevroletEngine, yellowBody, multifunctionedSteeringWheel, chevroletWheels));
-        carManager.carList.add(new Car("Ford", fordEngine, whiteBody, commonSteeringWheel, fordWheels));
-        carManager.carList.add(new Car("Audi", audiEngine, violetBody, commonSteeringWheel, audiWheels));
-        carManager.carList.add(new Car());
+        carManager.createCar("Mazda", mazdaEngine, blackBody, multifunctionedSteeringWheel, mazdaWheels, mazdaTank);
+        carManager.createCar("Chevrolet", chevroletEngine, yellowBody, multifunctionedSteeringWheel, chevroletWheels, chevroletTank);
+        carManager.createCar("Ford", fordEngine, whiteBody, commonSteeringWheel, fordWheels, fordTank);
+        carManager.createCar("Audi", audiEngine, violetBody, commonSteeringWheel, audiWheels, audiTank);
+        carManager.createDefaultCar();
```

Comment:
you could have addCar(Car car) method in CarManager and send a new or default car.
carManager.addCar(new Car);